package.json

```diff
@@ -50,6 +50,7 @@
     "proper-lockfile": "^4.1.1",
     "proxy-from-env": "^1.1.0",
     "rimraf": "^3.0.2",
+    "socksv5": "0.0.6",
```

Comment:
I looked over socksv5 deps, and I am not sure we would want that. I guess we'll have to implement it ourselves.